### ISSUE TITLE
The area behind an active tab is transparent

### ISSUE THREAD

**Issue by villor** [Adanos020/egui_dock#111]
**Describe the bug**
egui_dock 0.4.2

The area behind an active tab is not cleared with a background color.
This becomes an issue in bevy_editor_pls where a camera is projected behind egui and sometimes makes the text unreadable.

**To Reproduce**
1. Clone the [bevy_editor_pls](https://github.com/jakobhellermann/bevy_editor_pls) repo
2. If 0.4.2 is released, skip this step, otherwise add the following to crates/bevy_editor_pls_core Cargo.toml to use the transparency fix ([context](https://github.com/jakobhellermann/bevy_editor_pls/issues/68#issuecomment-1472850507))
`egui_dock = { git = "https://github.com/Adanos020/egui_dock.git", branch = "fix-clear-background" }` 
4. Run the basic example
`cd crates/bevy_editor_pls && cargo run --example basic`
5. Press E to open the editor view
6. See tabs

**Expected behavior**
The area behind the active tab should be cleared with a background color (like before 0.4). Or be configurable somehow.

**Screenshots**
![image](https://user-images.githubusercontent.com/7102243/225917048-8e677eec-7c3f-46c5-9ece-8799bd4de1dc.png)

**Context**
https://github.com/jakobhellermann/bevy_editor_pls/issues/68

**Comment by villor** [comment]
Actually I'm not sure if this is an issue with egui_dock at all.

It seems the default is to use `style.visuals.faint_bg_color` from the egui style for the `tab_bar_background_color`. 

Looking at the default egui dark theme it is set as:
`faint_bg_color: Color32::from_additive_luminance(5), // visible, but barely so`

I tried setting another color in `bevy_editor_pls` which removed the transparency.

```
--- a/crates/bevy_editor_pls_core/src/editor.rs
+++ b/crates/bevy_editor_pls_core/src/editor.rs
@@ -302,15 +302,19 @@ impl Editor {
             &mut internal_state.tree,
             egui_dock::Tree::new(Vec::default()),
         );
-        egui_dock::DockArea::new(&mut tree).show(
-            ctx,
-            &mut TabViewer {
-                editor: self,
-                internal_state,
-                world,
-                editor_state,
-            },
-        );
+        let mut style = egui_dock::Style::from_egui(&egui::Style::default());
+        style.tab_bar_background_color = egui::Color32::GOLD;
+        egui_dock::DockArea::new(&mut tree)
+            .style(style)
+            .show(
+                ctx,
+                &mut TabViewer {
+                    editor: self,
+                    internal_state,
+                    world,
+                    editor_state,
+                },
+            );
```

**Comment by Adanos020** [comment]
We'll keep an eye on this. If it causes more issues, we'll investigate.

**Comment by lvaroqui** [comment]
I personally think that the default is not correct, as a user I'd expect the whole DockArea to be opaque and opt in to transparency.

**Comment by Adanos020** [comment]
Closed by #119
